src/Php82/bootstrap.php

```diff
@@ -9,6 +9,20 @@
  * file that was distributed with this source code.
  */
 
+use Symfony\Polyfill\Php82 as p;
+
 if (\PHP_VERSION_ID >= 80200) {
     return;
 }
+
+if (!function_exists('odbc_connection_string_is_quoted')) {
```

Comment:
I think this should only be loaded if `odbc` extension is present.

Reply:
agreed